### DISCUSSION TITLE
Which is the last version not requiring Rust via its dependencies?

### DISCUSSION THREAD

**Discussion by barracuda156** [python-jsonschema/jsonschema#1211] [Q&A]
Last update of `jsonschema` in Macports has broken it for multiple systems due to introduction of Rust dependency.

Could someone please refer me to the last version which can be built without Rust?

**Comment by pajod** [comment]
The most recent release not depending on Rust is [v4.17.3](https://github.com/python-jsonschema/jsonschema/releases/tag/v4.17.3) which is also the last release to promise support for (end-of-life) Python 3.7.

`jsonschema` worked wherever Python worked until https://github.com/python-jsonschema/jsonschema/pull/1049 which was merged for the [4.18](https://github.com/python-jsonschema/jsonschema/releases/tag/v4.18.0) release.

Related: https://github.com/orgs/python-jsonschema/discussions/1118
